src/pybind/mgr/rook/ci/scripts/bootstrap-rook-cluster.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+
+set -eEx
+
+: ${CEPH_DEV_FOLDER:=${PWD}}
+KUBECTL="minikube kubectl --"
+
+# We build a local ceph image that contains the latest code
+# plus changes from the PR. This image will be used by the docker
+# running inside the minikube to start the different ceph pods
+LOCAL_CEPH_IMG="local/ceph"
+
+on_error() {
+    echo "on error"
+    minikube delete
+}
+
+cleanup() {
+    echo "cleanup"
+}
```

Comment:
the cleanup should delete the `minikube` and maybe prune the `docker` as well. Also the `tmp_build` folder which is created here, maybe that should also be removed?

Reply:
@nizamial09 this script has to create the minikube cluster so we can't delete it at the end because we haven't run the tests yet. Not sure if we should delete the minikube cluster or not. I expected that if everything is OK (with the tests) the vm gets deleted automatically but I'm not sure. I have to double check the ci guys.

Reply:
hmm, i thought the cleanup was run after the tests are completed/failed. Usually the script that does that stuffs needs to clean itself after. The jenkins machines are not cleared after every run.

Reply:
I added some steps in the cleanup to remove the minikube env